### PR TITLE
Add Bayesian Binary Sensor documentation

### PR DESCRIPTION
**Pull request by jlmcgehee21**
Why:

* I have a PR that is in progress on the main Hass repo.

This change addresses the need by:

* `binary_sensor.bayesian.markdown` in the `_components` directory.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8810

### CODE REVIEW

File: source/_components/binary_sensor.bayesian.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+description: "Instructions how to integrate threshold Bayesian sensors into Home Assistant."
+date: 2017-08-27 20:05
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.52
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.bayesian.markdown]: 0.53

File: source/_components/binary_sensor.bayesian.markdown [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+
+The `bayesian` binary sensor platform observes the state from multiple sensors
+and uses Bayes' rule to estimate the probability that an event has occurred
+given the state of the observed sensors. If the estimated posterior probability
+is above the `probabiliy_threshold`, the value of the sensor is `on`.
+Otherwise, the sensor is `off`.
+
+This allows for the detection of complex events that may not be readily
+observable, i.e. cooking, showering, in bed, start of morning routine, etc.  It
```
Comment (frenck) [inline source/_components/binary_sensor.bayesian.markdown]: **Spelling & Grammar**:
- Missing comma after `i.e.` -> `i.e., cooking, showering`
- `start of morning routine` -> `the start of a morning routine`

File: source/_components/binary_sensor.bayesian.markdown [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+can also be used to gain greater confidence about events that _are_ directly
+observable, but for which the sensors can be unreliable, i.e. presence.
```
Comment (frenck) [inline source/_components/binary_sensor.bayesian.markdown]: **Spelling & Grammar**:
- Missing comma after `i.e.` -> `unreliable, i.e., presence`

**Comment by jlmcgehee21** [comment]
:+1: :heart:

**Pull request merged**